README.md

```diff
@@ -58,14 +58,24 @@ eden eve console
 
 ## Run applications on EVE
 
-Deploy nginx server from dockerhub. Expose port 80 of the container to port 8081 of eve. 
+Notice: if you are on QEMU there is a limited number of exposed ports. Add some if you want to expose more.
+```
+hostfwd: 
+         {{ .DefaultSSHPort }}: 22 
+         5912: 5901 
+         5911: 5900 
+         8027: 8027 
+         8028: 8028
+``` 
+
+Deploy nginx server from dockerhub. Expose port 80 of the container to port 8028 of eve. 
 ``` 
-eden pod deploy -p 8081:80 docker://nginx
+eden pod deploy -p 8028:80 docker://nginx
 ``` 
 
-Deploy a VM from Openstack. Initialize root user with password - 'passw0rd'  Expose port 22 of the VM (ssh) to port 2222 of eve for ssh:
+Deploy a VM from Openstack. Initialize root user with password - 'passw0rd'  Expose port 22 of the VM (ssh) to port 8027 of eve for ssh:
 ``` 
-eden pod deploy -p 2222:22 http://cdimage.debian.org/cdimage/openstack/current/debian-10.4.3-20200610-openstack-arm64.qcow2 -v debug --metadata='#cloud-config\npassword: passw0rd\nchpasswd: { expire: False }\nssh_pwauth: True\n'
+eden pod deploy -p 8027:22 http://cdimage.debian.org/cdimage/openstack/current/debian-10.4.3-20200610-openstack-arm64.qcow2 -v debug --metadata='#cloud-config\npassword: passw0rd\nchpasswd: { expire: False }\nssh_pwauth: True\n'
```

Comment:
@mydatascience @giggsoff In this command, as I understand, we will have two variants:
* ...-arm64.qcow2 -- for Rpi
* ...-amd64.qcow2 -- for Qemu or Intel-based boards

Reply:
We have "arm64" version only in next PRs, so for now, only valid option is amd64.qcow2